server/routes/loan.js

```diff
@@ -25,11 +29,11 @@ const {
   checkGetRepayment,
   checkIdFormat,
 } = loanDataCheck;
-loan.post('/', loanApplyValidator, applicationCheck, apply);
-loan.get('/:loanId', checkIdFormat, checkGetLoan, getLoan);
-loan.patch('/:loanId', checkIdFormat, laonStatusValidator, checkStatus, changeStatus);
-loan.post('/:loanId/repayment', checkIdFormat, loanRepaymentValidator, postRepaymentCheck, postRepayment);
-loan.get('/:loanId/repayments', checkIdFormat, checkGetRepayment, getRepayHistory);
-loan.get('/', checkQueryStrings, getLoans);
+loan.post('/', loanApplyValidator, applicationCheck, checkToken, isUser, apply);
+loan.get('/:loanId', checkIdFormat, checkGetLoan, checkToken, isAdmin, getLoan);
+loan.patch('/:loanId', checkIdFormat, laonStatusValidator, checkStatus, checkToken, isAdmin, changeStatus);
```

Comment:
You have a typo in here ```laonStatusValidator```, I recon you meant ```loanStatusValidator```. Please rename this middleware.

Reply:
okay @damafeez